bin/init.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 export ISTIO_REMOTE=${ISTIO_REMOTE:-origin}
 export ISTIO_BRANCH=${ISTIO_BRANCH:-master}
 
+echo "Updating the istio/istio dependency to the latest ${ISTIO_BRANCH} branch ..."
+go get istio.io/istio@${ISTIO_BRANCH}
```

Comment:
I don't think we want to do this automatically? Since then what is checked in and what is tested are two different things?

Reply:
I suspect that we will need to add something in the branching instructions, but I would think that currently we would want to be testing against the `release-1.5` branch (set when the release branch is created for istio/istio). Once we created the 1.5 doc branch we would revert to testing against `master` (release-1.5 istio.io branch continues to get against 1.5).